model_compression_toolkit/gptq/keras/gptq_training.py

```diff
@@ -133,28 +133,21 @@ def _is_gptq_applicable(self,
 
     def gptq_wrapper(self,
                      n: common.BaseNode,
-                     layer: Layer,
-                     include_activation_quantizers: bool = False) -> Union[qi.KerasQuantizationWrapper, Layer]:
+                     layer: Layer) -> Union[qi.KerasQuantizationWrapper, Layer]:
         """
         A function which takes a computational graph node and a keras layer and perform the quantization wrapping.
 
         Args:
-            include_activation_quantizers: Whether to use the wrapper for the activation quantizer or not
             n: A node of mct graph.
             layer: A keras layer
 
         Returns: Wrapped layer if the layer should be wrap, otherwise returns the layer as is.
 
         """
-        if self._is_gptq_applicable(n):
+        if self._is_gptq_weights_trainable(n):
             weights_quantizers, activation_quantizers = quantization_builder(n, self.gptq_config)
```

Comment:
maybe replace activation_quantizers  with '_' since this variable is not being used (and it can be confusing)